socorro/unittest/processor/test_signature_utilities.py

```diff
@@ -50,6 +50,7 @@ def setup_config_C_sig_tool(
         pr='pre1|pre2',
         si='fnNeedNumber',
         ss=('sentinel', ('sentinel2', lambda x: 'ff' in x)),
+        ellipsis_arguments=False
```

Comment:
Why is this an argument? It's never set to true. The `config.collapse_arguments` is hardcoded to True.